src/lecturer/services/StoreService.tsx

```diff
@@ -1,12 +1,51 @@
-import { createContext, ReactNode, useState } from "react";
+import { createContext, ReactNode, useEffect, useState } from "react";
 
 export interface StoreProps {
     children: ReactNode
 }
 
+enum StorageKey {
```

Comment:
why did you decide to use enum here?

Reply:
To get autocompletion from typescript (xD) and to be sure that a don't repeat any key.